source/client-side-encryption/client-side-encryption.rst

```diff
@@ -773,10 +811,34 @@ masterKey contents
 
 ======== ======== ========================================================================
 **Name** **Type** **Description**
-provider String   Either "aws" or "local". More providers will be available in the future.
+provider "aws"    -
 key      String   AWS ARN. Only applicable for "aws" provider.
 region   String   AWS Region that contains AWS ARN. Only applicable for "aws" provider.
-endpoint string   Alternate AWS endpoint (needed for FIPS endpoints)
+endpoint String   Alternate AWS endpoint (needed for FIPS endpoints)
+======== ======== ========================================================================
+
+================= ======== ===============================================================
+**Name**          **Type** **Description**
+provider          "azure"  -
```

Comment:
Strangely, the `-` here renders as a bullet point in the Github view. Maybe it needs to be escaped somehow?